jenkins/release-workflows/release-manifest-commit-lock.jenkinsfile

```diff
@@ -155,7 +155,11 @@ pipeline {
                         selectedComponents.each { componentName ->
                             def existingComponent = existingManifest.components.find { it.name == componentName.name }
                             if (existingComponent) {
-                                def releaseTag = params.RELEASE_VERSION + ".0"
+                                def versionTokenize = params.RELEASE_VERSION.tokenize('-')
+                                def releaseTag = versionTokenize[0] + ".0"
+                                if (versionTokenize.size() > 1) {
+                                    releaseTag = versionTokenize[0] + ".0-" + versionTokenize[1]
+                                }
                                 if (componentName.name == 'OpenSearch' || componentName.name == 'OpenSearch-Dashboards' || componentName.name == 'functionalTestDashboards') {
                                     releaseTag = params.RELEASE_VERSION
```

Comment:
I think this also needs the changes? For cores 3.0.0-alpha1?

Reply:
Release param will be 3.0.0-alpha1 so it wont tokenize and will remain same. Tested with and without qualifier

Reply:
Oh I get what you mean now. Cool.